src/test_harness/bitcoin.rs

```diff
@@ -40,6 +40,8 @@ impl<'c> Blockchain<'c> {
             .create_wallet(&self.wallet_name, None, None, None, None)
             .await?;
 
+        bitcoind_client.rescan(&self.wallet_name).await?;
```

Comment:
How can there be founding a new random wallet? This is useless.

Reply:
removed.